docs/application/dotnet/guides/app-management/widget-viewer.md

```diff
@@ -0,0 +1,133 @@
+# Widget viewer
+
+[Widgets](/application/dotnet/guides/applications/widget-app.md) are commonly used in applications like home screen (launcher) or lock screen. **Widget** is a simple component based on NUI View designed to show personalized set of information.
+
+**NUI** Framework provides API to use and view widgets in your application. To read more about widget application you can check:
+- [Widget Application Guide](/application/dotnet/guides/applications/widget-app.md)
+- [Widget Management Guide](./widget-control.md)
+
+Platform applications preinstalled in the Tizen image provides few simple widgets such as `Gallery`, `Music Player`, and `Contacts`. To check installed widgets application in your Tizen image, you can use sdb tool and **pkginfo** command line interface.
+
+To open device / emulator shell please use: 
+
+```shell
+sdb shell
+```
+
+Then list installed widgets:
+
+```shell
+pkginfo --listpkg | grep widget
+```
+
+## Widget viewer sample
+
+The code snippet below implements simple Widget Viewer application.
+
+**Figure: Widget Viewer**
+
+![Widget Viewer](./media/widget_viewer.png)
+
+### Prerequisites
+To view a widget, the application has to request permission by adding the following privilege to the  `tizen-manifest.xml` file:
+
+    ```
+    <ui-application appid="org.tizen.example.NUIWidgetViewer">
+        <!-- application description -->
+    </ui-application>
+    <privileges>
+       <privilege>http://tizen.org/privilege/widget.viewer</privilege>
+    </privileges>
+    ```
+
+### Code snippet
```

Comment:
As per my comment provided earlier (please see below), could you please update the heading in this topic?

In reference to the comment provided in PR#1503 (https://github.com/Samsung/tizen-docs/pull/1503/files#r695509286), if you are planning to fix/rephrase the heading "Code Snippet" in that PR, could you please update here as well?

Reply:
For this MD file the format is slightly different from 1503 and 1504 so we can either retain code snippet title as is.
Or change it to
### Steps

@mijin85cho your thoughts?

Reply:
@annie-samsung You can choose which way to manage the headings.

Well, if possible, following the same format as like other pages would be better... For example as like

\## Widget viewer application 

\## Prerequisites

\## Create widget viewer

\## Manage widget viewer

Reply:
@mijin85cho thanks I think your suggestion is good. @sagarkedari, kindly verify if the same has been applied when the developer fixes this topic.